rstudio/c9s-python-3.9/Dockerfile

```diff
@@ -50,6 +50,7 @@ COPY rsession.conf /etc/rstudio/rsession.conf
 # package installation 
 RUN dnf install -y libsodium-devel.x86_64 libgit2-devel.x86_64 libcurl-devel harfbuzz-devel.x86_64 fribidi-devel.x86_64 cmake "flexiblas-*"
 RUN R -e "install.packages('Rcpp')"
+RUN R -e 'install.packages(c("tidyverse", "tidymodels", "plumber", "vetiver", "devtools"))'	
```

Comment:
c.f. [RHOAIENG-2799](https://issues.redhat.com/browse/RHOAIENG-2799) R packages preinstalled in the R Studio image are not version-locked